docs/UserGuide.md

```diff
@@ -200,26 +200,98 @@ Format: `delete_hairdresser ID`
 * The index **must be a positive integer** 1, 2, 3, …​
 * All appointments with this corresponding hairdresser will now show "DELETED" for the hairdresser placeholder
 
+
 ### Alias commands
+(Contributed by Aloysius)
+
+This feature allows you to customise aliases. You can add, delete and list aliases which are shortcuts for 
+longer commands.
+
+
+
+
+
+#### Add a new alias : `add_alias`
+
+You can use this command to add a new alias to the program.
+
+*Format:*
+
+`add_alias old/OLD_ALIAS new/NEW_ALIAS`
+
+*Example:*
+
+You want to create an alias `aa` for `add_alias`. 
+
+****
+Creating a new alias:
+
+. Type `add_alias old/add_alias new/aa` into the _Command Box_.
+. Press `Enter` to execute.
+
+Outcome:
+
+. The _Result Display_ will show a success message.
+. You can now see your new alias
+****
+
+.Outcome of a successful `add_alias` command
+![Ui](images/add_alias_success.png)
+
+***
+
+You can now use the alias
+![Ui](images/use_new_alias.png)
+
+***
+
+And it works!
+![Ui](images/new_alias.png)
+
+[NOTE]
```

Comment:
In the md syntax, we should use <div> tags to achieve similar callout instead. Refer to the callout in lines 31 - 47 to see how it was done under Features.